.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve our operating system.
+title: '[BUG] Brief description of the problem'
```

Comment:
I am not sure whether having every issue starting its title with `[Category]` is a good idea. Have you seen any established projects following this practice? I would like to see how it works in practice. I have three concerns with this convention. First, the same information is already presented by the `C-` labels. Thus, presenting this information in the issue titles again is redundant. Second, it is kind of verbose for every issue title to include `[Category]`. Human audience have no problems at all regarding which category an issue belongs to simply by looking at its title. Third, some people may prefer prefixing the issue title with `[Subsystem]`. If we force every issue title to start with `[Category]`, then people may have to write `[Category][Subsystem]`, which is kind of ugly.

Reply:
The effect of current PR can be seen in [grief/asterinas](https://github.com/grief8/asterinas/issues/new?assignees=&labels=C-tracking-issue&projects=&template=tracking_issue.md&title=%5BTRACKING%5D+Short+summary+of+the+task+or+feature). `[Category]` in title is editable but it indeed guides the developers to name their issues with `[Category]`. So I agree to delete it. 

Actually [rust-lang](https://forge.rust-lang.org/release/issue-triaging.html#labels) follows such style with `C-`, and I prefer it to `[Category]`.